### ISSUE TITLE
Image not clicking in way point mission (P4P)

### ISSUE THREAD

**Issue by rohitrameshrao** [dji-sdk/Mobile-SDK-Android#261]
In waypoint mode, when specifying to take images at a specified time interval, images are not getting clicked for a brief duration sometimes.  This is creating a huge problem for our users who are not able to use our app reliably. I believe this is a firmware issue. When can we expect a fix? This is super critical and fundamental functionality of any mission planning app.

SDK version: 4.3.2
Drone: Phantom 4 Pro
Image interval: 5 sec
SD card: Class 10
Android: 6.0.1
Device: Samsung Galaxy J7 Prime

**Comment by Michael-DJI** [comment]
yes, it's a firmware bug, we will release a new firmware to fix it! thanks for the feedback.
